opencensus-shim/build.gradle.kts

```diff
@@ -21,4 +21,12 @@ dependencies {
 
   testImplementation("org.slf4j:slf4j-simple")
   testImplementation("io.opencensus:opencensus-impl")
+  testImplementation("io.opencensus:opencensus-contrib-exemplar-util")
+}
+
+tasks.named<Test>("test") {
+  // We must force a fork per-test class because OpenCensus pollutes globals with no restorative
```

Comment:
Is it possible to split test sets to work around this? forkEvery(1) goes sloooow

Reply:
Only for some of the tests.  I've tried to limit what's there to just the ones we need, but for some of the coverage I do needs to actually set up OpenCensus.

OpenCensus itself tended to use heavy mocks/mocking for various metric things.  Really appreciate in OTel the global-cleaning utiities.

Reply:
Yeah if we can identify those tests that need it and split them into test suites at least it doesn't have to fork for every single one.